src/main/java/by/stetsko/operation/AbstractOperation.java

```diff
@@ -0,0 +1,4 @@
+package by.stetsko.operation;
+
+public abstract class AbstractOperation {
```

Comment:
Use interface in combination with abstract class
like
_ReverseWordOrderOperation extends AbstractOperation
AbstractOperation implements IOperation
IOperation {
    String process();
}_